payments-core/src/test/java/com/stripe/android/GooglePayJsonFactoryTest.kt

```diff
@@ -32,7 +32,9 @@ class GooglePayJsonFactoryTest {
                     "type": "CARD",
                     "parameters": {
                         "allowedAuthMethods": ["PAN_ONLY", "CRYPTOGRAM_3DS"],
-                        "allowedCardNetworks": ["AMEX", "DISCOVER", "MASTERCARD", "VISA"]
+                        "allowedCardNetworks": ["AMEX", "DISCOVER", "MASTERCARD", "VISA"],
+                        "allowCreditCards": true,
```

Comment:
This is my only FUD with this PR. 

Can you manually test to make sure no unexpected behavior changes happen as a result of us sending true here.